docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,228 @@
+combination of short bucket span and high cardinality is a resource-intensive 
+situation that you may want to avoid.
+
+
+[[population-analysis]]
+=== Using population analysis
+
+Population analysis is more memory efficient than individual analysis of each 
+series. It builds a profile of what a "typical" entity does over a specified 
+time period and then identifies when one is behaving abnormally compared to the 
+population. Use population analysis for analyzing high cardinality fields if you 
+expect that the entities of the population generally behave in the same way.
+
+
+[[pre-aggregating-data]]
+=== Pre-aggregating data
+
+{anomaly-jobs-cap} always work on summary statistics calculated for the buckets. 
+The statistics can be calculated in the job itself or via aggregations. It is 
+more efficient to use an aggregation when it's possible, as in this case, the 
+data node does the heavy-lifting instead of the {ml} node.
+
+In certain cases, you cannot do aggregations to increase performance. For 
+example, categorization jobs use the full log message to detect anomalies, so 
+this data cannot be aggregated. If you have many influencer fields, it may not 
+be beneficial to use an aggregation either, because only a few documents in each 
+bucket may have the combination of all the different influencer fields.
+
+Consult <<ml-configuring-aggregation>> to learn more.
+
+
+[[disabling-model-plot]]
+=== Disabling model plot
+
+Model plot calculates and stores the model bounds for each analyzed entity. It 
+generates a huge amount of results and has a significant impact on performance. 
+The indexing load is the same as for forecasting: one document per bucket per 
+split field value. If you have high cardinality fields and/or short bucket span, 
+disable model plot if it's not needed.
+
+
+[[results-retention]]
+=== Setting result retention time
+
+{anomaly-detect-cap} results are retained indefinitely by default. For big jobs, 
+results build up over time, and your result index will be quite large. Consider 
+how long you wish to retain the results and set `results_retention_days` 
+accordingly – for example, to 30 or 60 days – to avoid unnecessarily large 
+result indices. Deleting old results does not affect the model behavior.
+
+
+[[renormalization]]
+=== Configuring renormalization window days
+
+When a new anomaly has a much higher score than any anomaly in the past, the 
+anomaly scores are adjusted on a range from 0 to 100 based on the new data. This 
+is called renormalization. It can mean rewriting a large number of documents in 
+the results index. Renormalization happens for results from the last 30 days or 
+100 bucket spans (depending on which is the longer) by default. When you are 
+working at scale, set `renormalization_window_days` to a lower value, so the 
+workload is reduced.
+
+
+[[model-snapshots]]
+=== Setting model snapshot retention time 
+
```

Comment:
This entire section does not seem to be giving quite the right advice for scale users - plus its confusing, which isn't necessarily a docs fault. It tries to explain what each setting does, but not the guidelines and trade-offs at scale.

This advice would only be for jobs that have large models. Check this first.

We should explain that reducing the frequency of  `background_persist_interval` reduces work load, but means that the job may restore older model state in the event of a node failure.

We should explain that setting `model_snapshot_retention_days` to a smaller value reduces the size of the model state stored on disk. But means that you have a smaller window of time from which you can restore a model. 

Jobs created in 7.8 and above, already have `daily_model_snapshot_retention_after_days` set to 1. Consider setting this to 1 for jobs created prior to 7.8.